server/test/red-flag.spec.js

```diff
@@ -45,6 +52,7 @@ describe('Ride-My-Way', () => {
       const redFlag = db[2];
       const response = await server.patch(`/api/v1/red-flags/${redFlag.id}/location`);
       expect(response.body.status).to.equal(200);
+      expect(response.body.data[0].message).to.eql('Updated red-flag record\'s location');
```

Comment:
You should also check that the updated data is returned with the updated values.